views/searchResults.hbs

```diff
@@ -0,0 +1,14 @@
+<div id="search">
+
+  <p>Albums & Singles by:</p>
```

Comment:
careful with & and other special chars in HTML -> sometimes they break, there is syntax for escaping it though! 
https://stackoverflow.com/questions/7381974/which-characters-need-to-be-escaped-in-html